upload-blocks/BlockComponent/BlockComponent.js

```diff
@@ -13,9 +13,11 @@ if (!DOC_READY) {
   });
 }
 
-let externalPropsAdded = false;
-
 export class BlockComponent extends BaseComponent {
+  /**
+   * @param {String} str
```

Comment:
Why `String` instead of `string`? 

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#general-types

Reply:
Because JavaScript type is String and there is no string... I think we could step from the TS recommendations here...